src/Dapr.Client/GRPCStatusInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+namespace Dapr.Client
+{
+    using Dapr.Client.Http;
+    using System;
+    using System.Net;
+
+    /// <summary>
+    /// This class is only needed if the app you are calling app is listening on gRPC.
+    /// It contains propertes that represent status info that may be populated for an gRPC response.
+    /// </summary>
+    public class GRPCStatusInfo
```

Comment:
The .NET naming convention for this would be `GrpcStatusInfo`. This would also be consistent with how the .NET gRPC APIs are named (see line 20).